PF1/Assignment2/RemoveDuplicate.java

```diff
@@ -0,0 +1,34 @@
+package assignment2;
+// Program to remove duplicate element from array by assigning max value to duplicate elements
+public class RemoveDuplicate {
+	 int[] removeDuplicate(int input[])
+	 {
+		 int max=Integer.MAX_VALUE ;
+		
+	   // assigning max value of integer to duplicate element in array
```

Comment:
Try to implement with logic you told in review meeting, This is having too much complexity and will fail in some cases like array collection may have zero or max value of integer